RatingsFilms/RatingsFilms/NetworkService.swift

```diff
@@ -0,0 +1,93 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Type of errors
+enum NetworkError: Error {
```

Comment:
```suggestion
enum NetworkError: Error {
```
Можно использовать стандартную ошибку Error